services/background-worker/test/Test/Wire/BackendNotificationPusherSpec.hs

```diff
@@ -19,122 +40,280 @@ import Wire.API.Federation.BackendNotifications
 import Wire.API.RawJson
 import Wire.BackendNotificationPusher
 import Wire.BackgroundWorker.Env
+import Wire.BackgroundWorker.Options
 
 runTestAppT :: AppT IO a -> Int -> IO a
 runTestAppT app port = do
   http2Manager <- initHttp2Manager
   logger <- Logger.new Logger.defSettings
   let federatorInternal = Endpoint "localhost" (fromIntegral port)
+      rabbitmqAdminClient = undefined
+      rabbitmqVHost = undefined
```

Comment:
I guess `undefined` is fine here?

Reply:
Yeah, I am trying to write tests without polysemy, I think this is the easiest thing we can do.